File: C/program70.c
#include<stdio.h>

void DisplayEven(int iNo)
{
    if(iNo < 0)
    {
        iNo = - iNo;
    }
    
    int iCount =0, iDigit = 0;

    printf("Even Digits are :\n");
    while(iNo > 0)
    {   
        iDigit = iNo % 10;
        
        if((iDigit % 2) == 0)
        {
            printf("%d\n",iDigit);
        }

       
        iNo = iNo / 10;
        
    }
    
   
}

int main()
{
    int iValue = 0;

    printf("Enter number :\n");
    scanf("%d",&iValue);

    DisplayEven(iValue);

    return 0;
}

File: Python/program70.py
class Digit:
    _No = 0
    
    def __init__(self, iNo):
        self._No = iNo
           
    def Display_Even_Digits(self):
        
        if(self._No < 0):
            self._No = -self._No
        print("Even digits are ")
        while(self._No > 0):
            iDigit = self._No % 10
            if((iDigit % 2) == 0):
                print(iDigit)
            self._No = self._No // 10
            
       
    
def main(): 
    
    iValue = int (input("Enter number : "))
        
    sobj = Digit(iValue)
    sobj.Display_Even_Digits()
    
if __name__ == "__main__":
    main()